File: JY-CH/baekjoon/b28702.py
# FizzBuzz
import sys
input = sys.stdin.readline


check = ['Fizz', 'Buzz', 'FizzBuzz']
# 피즈버즈
# 3개씩 보는데 동시에 3개가 저 아래 체크를 완성할 수 없음.
# 무슨 짓을해도 n은 결국 동일한 값이 나옴
for i in range(3):
    x = input().strip()
    if x not in check:
        n = int(x) + (3 - i)

if n % 3 == 0 and n % 5 == 0:
    print('FizzBuzz')
elif n % 3 == 0:
    print('Fizz')
elif n % 5 == 0:
    print('Buzz')
else:
    print(n)

File: dukcode/solvedac/class_02/B28702.java
package com.dukcode.solvedac.class_02;

import java.io.BufferedReader;
import java.io.BufferedWriter;
import java.io.IOException;
import java.io.InputStreamReader;
import java.io.OutputStreamWriter;

public class B28702 {

  private static BufferedReader br;
  private static BufferedWriter bw;

  public static void main(String[] args) throws IOException {
    br = new BufferedReader(new InputStreamReader(System.in));
    bw = new BufferedWriter(new OutputStreamWriter(System.out));

    bw.write(getAns(getNumber()));

    br.close();
    bw.close();

  }

  private static int getNumber() throws IOException {
    for (int i = 0; i < 3; ++i) {
      String str = br.readLine();
      if (isNumber(str)) {
        return Integer.parseInt(str) + (3 - i);
      }
    }

    return -1;
  }

  private static boolean isNumber(String s) {
    return !s.endsWith("z");
  }

  private static String getAns(int num) {
    if (num % 3 == 0 && num % 5 == 0) {
      return "FizzBuzz";
    }

    if (num % 3 == 0) {
      return "Fizz";
    }

    if (num % 5 == 0) {
      return "Buzz";
    }

    return String.valueOf(num);
  }

}
